src/app/games/epaminondas/EpaminondasTutorial.ts

```diff
@@ -0,0 +1,128 @@
+            if (move.movedPieces === 1) {
+                return MGPValidation.SUCCESS;
+            } else {
+                return MGPValidation.failure($localize`Congratulations, you are in advance. But this is not the exercise here, try again.`);
+            }
+        },
+        $localize`This is how you move a single piece.`,
+    ),
+    TutorialStep.fromPredicate(
+        $localize`Moving a phalanx`,
+        $localize`Let us now see how to move multiple pieces along a line, which is called a phalanx:
+        <ol>
+            <li>Click on the first piece of the phalanx.</li>
+            <li>Click on the last piece of the phalanx.</li>
+            <li>Click on one of the squares highlighted in yellow; you can move your phalanx up to a distance equal to its length.</li>
+        </ol><br/>
+        Move a phalanx!`,
+        EpaminondasPartSlice.getInitialSlice(),
+        new EpaminondasMove(0, 11, 2, 1, Direction.UP),
+        (move: EpaminondasMove, _: EpaminondasPartSlice) => {
+            if (move.movedPieces > 1) {
+                return MGPValidation.SUCCESS;
+            } else {
+                return MGPValidation.failure($localize`Failed! You moved only one piece.`);
+            }
+        },
+        $localize`Congratulations!
+        The moved pieces can be horizontally, vertically, or diagonally aligned.
+        The move should be made along this axis, forward or backwards.
+        There can be no opponent nor holes in the phalanx.`,
+    ),
+    TutorialStep.fromMove(
+        $localize`Capture`,
+        $localize`In order to capture pieces of the opponent:
+        <ol>
+            <li>It must be aligned with the phalanx you are moving.</li>
+            <li>It must be strictly shorter than your phalanx.</li>
+            <li>The first piece of your phalanx should land on the first piece of the opponent's phalanx that you want to capture.</li>
+        </ol><br/>
+        Capture a phalanx.`,
+        new EpaminondasPartSlice([
+            [_, _, _, _, _, _, _, _, X, X, X, X, X, X],
+            [_, _, _, _, _, _, X, _, _, _, _, _, _, _],
+            [_, _, _, _, _, X, _, _, _, _, _, _, _, _],
+            [_, _, X, _, X, _, _, _, _, _, _, _, _, _],
+            [_, _, X, X, _, _, _, _, _, _, _, _, _, _],
+            [_, _, X, _, _, _, _, _, _, _, _, _, _, _],
+            [_, _, _, _, _, _, _, _, _, _, _, _, _, _],
+            [_, _, O, _, _, _, _, _, _, _, _, _, _, _],
+            [_, _, O, _, _, _, _, X, _, _, _, _, _, _],
+            [_, _, O, _, _, _, _, _, _, _, _, _, _, _],
+            [_, _, O, _, _, _, _, _, _, _, _, _, _, _],
+            [_, _, _, _, _, _, _, O, O, O, O, O, O, O],
+        ], 0),
+        [new EpaminondasMove(2, 10, 4, 2, Direction.UP)],
+        $localize`Congratulations, you made it!
+        Note that the diagonal phalanx was not aligned with your phalanx, hence even if it is longer, this does not prevent you from capturing some of its pieces on another alignment.`,
```

Comment:
although instead of hence ?

Reply:
la diagonale n'était pas alignée, bien que même si elle est plus longue, ça ne l'empêche pas [...]
vs.
la diagonale n'était pas alignée, donc même si elle est plus longue, ça ne l'empêche pas [...]

Reply:
En français, "bien que" clairement, puisqu'on vient expliquer une subtilité/"exception"

Reply:
bien que : "exprime la concession, la reconnaissance d'un fait qui devrait empêcher un autre fait de se réaliser, mais ne l'empêche pas. Exemple : Normalement, je devrais obtenir une bonne note à cet examen de mathématiques, bien que je ne sois pas sûr d'avoir tous les points au dernier exercice."
Ici c'est pas trop le cas non ?